modules/swagger-codegen/src/main/resources/csharp/ApiClient.mustache

```diff
@@ -145,7 +145,7 @@ namespace {{packageName}}.Client
         /// <param name="contentType">Content Type of the request</param>
         /// <returns>Object</returns>
         public Object CallApi(
-            String path, RestSharp.Method method, Dictionary<String, String> queryParams, Object postBody,
+            String path, RestSharp.Method method, List<KeyValuePair<String, String>> queryParams, Object postBody,
```

Comment:
On second thought, because every key in a Dictionary must be unique, I don't think it is the appropriate type to be using for query parameters.